src/data/faq.json

```
@@ -2545,6 +2545,11 @@
                     "term": "iOS",
                     "anchor": "ios",
                     "description": "iOS is a mobile operating system developed by Apple for the iPhone, iPod touch and formerly also iPad. Along with <a href='#glossary_android'>Android</a>, it is one of the two operating systems that the Corona-Warn-App supports."
+                },{
+                    "term": "Isolation",
+                    "anchor": "isolation",
+                    "description": "Isolation is a government-ordered measure for persons who have been confirmed to have SARS-CoV-2 infection by a PCR-Test.Isolation can be gone through at home or, if the disease is severe, in a hospital. Release from isolation is based on specified <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Entlassmanagement.html?nn=13490888'> criteria (German only)</a> . At the time of release, the person is considered to no longer be infectious. See also <a href='#glossary_quarantine'>Quarantine</a>. "

```

Whitespace corrections:

```suggestion
                    "description": "Isolation is a government-ordered measure for persons who have been confirmed to have SARS-CoV-2 infection by a PCR-Test. Isolation can be gone through at home or, if the disease is severe, in a hospital. Release from isolation is based on specified <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Entlassmanagement.html?nn=13490888'> criteria (German only)</a>. At the time of release, the person is considered to no longer be infectious. See also <a href='#glossary_quarantine'>Quarantine</a>. "
```